packages/vite/src/node/build.ts

```diff
@@ -871,6 +871,17 @@ export function toOutputFilePathInString(
   return config.base + filename
 }
 
+function toImportMetaURLBasedRelativePath(
+  filename: string,
+  importer: string
+): { runtime: string } {
+  return {
+    runtime: `new URL(${JSON.stringify(
+      path.posix.relative(path.dirname(importer), filename)
+    )},import.meta.url).href`
```

Comment:
Nit:

```suggestion
    )}, import.meta.url).href`
```

Reply:
Oh. Is the return value supposed to be minified output, where that space is intentionally omitted?

Reply:
Actually we should review all these if we do in a separate PR (there are several snippets like this in the code base)